Gopkg.lock

```diff
@@ -293,19 +293,6 @@
   pruneopts = ""
   revision = "e89373fe6b4a7413d7acd6da1725b83ef713e6e4"
 
-[[projects]]
```

Comment:
This is independent change. not sure why this is in this pr. consider adding a separate commit for this.

Reply:
This change is probably not needed once I add some tests back in.